package.json

```diff
@@ -8,6 +8,11 @@
     "build": "yarn tsc -p tsconfig.build.json",
     "test": "yarn jest"
   },
+  "files": [
```

Comment:
This means the files being published, we won't remove lib from the final tarball

Reply:
Ops! You are right!